scripts/visualize/visualizeWalleyeDecline.R

```diff
@@ -20,7 +20,7 @@ svgFishTrends <- function(object, filename){
   dinosvg:::write_svg(svg, file = filename)
 }
 
-visualize.visualizeFishTrends <- function(item){
+visualize.visualizeFishTrends <- function(outfile, ...){
   fish <- readData(item$filename)
```

Comment:
this function can+should expect arguments named for the IDs specified in the `depends` field for the figure items in viz.yaml. that's how you can tell them apart. (one call will include `processedWallyTrends=readData('processedWallyTrends')` as an argument; the other will include `processedBassTrends=readData('processedBassTrends')`